### PR TITLE
testCases

### PR DESCRIPTION
**Pull request by olaleks**
Screen Shot 2022-05-02 at 9 56 29 PM

### CODE REVIEW

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        $("#city").sendKeys("Miami");
+        $("#id_state").selectOptionByValue("9");
+        $("#postcode").sendKeys("33131");
+        $("#id_country").selectOptionByValue("21");
+        $("#other").sendKeys("Additional Information");
+        $("#phone").sendKeys("+375292929297");
+        $("#phone_mobile").sendKeys("+375333333333");
+        $("#alias").sendKeys("18, Baker street");
+        $("#submitAccount").click();
+        sleep(10000);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete this or change to waiter

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        $("a[title=\"Contact us\"]").click();
+        $("#id_contact").selectOptionByValue("2");
+        $("#uniform-id_contact").click();
+        $("div.selector [name=\"id_order\"]").selectOptionContainingText(orderNumber);
+        String orderId = $("div.selector [name=\"id_order\"]").getValue();
+        $("select[name=\"id_product\"][id=\""+ orderId+ "_order_products\"]").selectOption(1);
+        $("#message").sendKeys("I have a problem with my order. Could you help me?");
+        $("#submitMessage").click();
+        $("a.logout[title=\"Log me out\"]").click();
+        sleep(20000);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete this

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        Matcher matcher = orderNumberPattern.matcher(text);
+        Assertions.assertTrue(matcher.find());
+        String orderNumber = matcher.group(0);
+
+        $("a[title=\"Contact us\"]").click();
+        $("#id_contact").selectOptionByValue("2");
+        $("#uniform-id_contact").click();
+        $("div.selector [name=\"id_order\"]").selectOptionContainingText(orderNumber);
+        String orderId = $("div.selector [name=\"id_order\"]").getValue();
+        $("select[name=\"id_product\"][id=\""+ orderId+ "_order_products\"]").selectOption(1);
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java:94]: excellent

**Review by Kirill-Simonovich — APPROVED** [review]

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+    }
+    @Test
+    public void testCaseOrderDress(){
+        open("http://automationpractice.com/index.php");
+        $(".login").click();
+        $("#email").sendKeys("test54321@yopmail.com");
+        $("#passwd").sendKeys("12345");
+        $("#SubmitLogin").click();
+        $("ul.sf-menu>li>a[title=\"Dresses\"]").hover();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: .sf-menu > li > [title='Dresses'] better

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        $("ul.submenu-container>li>a[title=\"Casual Dresses\"]").click();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: .submenu-container > li > [title='Casual Dresses']

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+        $$("div.product-container a.product_img_link[title=\"Printed Dress\"]").first().hover();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: .product-container .product_img_link[title='Printed Dress']
delete tags

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+        $("div.button-container a.button[data-id-product=\"3\"]").click();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete tag

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+        $("a.btn[title=\"Proceed to checkout\"]").click();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete tag

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+        $("p.cart_navigation>a[title=\"Proceed to checkout\"]").click();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete tag

File: src/test/java/com/stormnet/TestCasesAutomation.java [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+        $("p.cart_navigation>button[name=\"processAddress\"]").click();
```
Comment (Kirill-Simonovich) [inline src/test/java/com/stormnet/TestCasesAutomation.java]: delete tag

**Comment by Kirill-Simonovich** [comment]
delete unnecessary tags In CSS selectors

**Review by Kirill-Simonovich — CHANGES_REQUESTED** [review]

**Comment by olaleks** [comment]
Screen Shot 2022-05-10 at 9 56 39 AM

File: src/test/java/com/stormnet/TestCasesAutomation.java (hunk 10 above)
Comment (olaleks) [reply src/test/java/com/stormnet/TestCasesAutomation.java]: Deleted all unnecessary tags, added 2 rest cases. Also I added configuration. But I don't understand where it need to be - create package or create separate class?

**Review by Kirill-Simonovich — APPROVED** [review]

**Pull request merged**